test/common/stats/stat_test_utility.cc

```diff
@@ -135,6 +135,37 @@ MemoryTest::Mode MemoryTest::mode() {
 #endif
 }
 
+// TODO(jmarantz): this utility is intended to be costs both for unit tests
```

Comment:
"costs both"? Feel free to fix in a follow up.

Reply:
Thanks Matt, rolling that into #9355